### PR TITLE
YaST-related parts for 389-ds ldap server

### PR DESCRIPTION
**Pull request by taroth21**
### Description
Working with @Firstyear to finish the YaST parts of the 389 ds server documentation.  

### Checklist

*Are backports required?*

- [x] To maintenance/SLE15SP1
- [x] To maintenance/SLE15SP0
- [ ] To maintenance/SLE12SP5
- [ ] To maintenance/SLE12SP4
- [ ] To maintenance/SLE12SP3

### CODE REVIEW

File: xml/security_ldap.xml [hunk 1]
```diff
@@ -784,7 +784,95 @@ systemctl start sssd</screen>
...
+   </informalfigure>
+  </step>
+  <step>
+   <para>
+    If all setting are according to your wishes, confirm with <guimenu>OK</guimenu>.
+   </para>
+  </step>
+ </procedure>
+ <para>
+  <remark>taroth 2020-02-20: @firstyear: does another window follow if I click 'OK'?
```
Comment (Firstyear) [inline xml/security_ldap.xml]: There are no following windows, it just installs

File: xml/security_ldap.xml [hunk 2]
```diff
@@ -784,7 +784,95 @@ systemctl start sssd</screen>
...
+   I could not test it  as I do not have test certificates (and one cannot click
+   'OK' without filling all fields, which makes sense)</remark>
```
Comment (Firstyear) [inline xml/security_ldap.xml]: That must be the old version, the new version doesn't require certificates ....

File: xml/security_ldap.xml [hunk 3]
```diff
@@ -784,7 +784,95 @@ systemctl start sssd</screen>
...
+  &yast; creates a directory server instance with the settings you have specified.
+  <remark>taroth 2020-01-20: @firstyear: what happens in the background?
```
Comment (Firstyear) [inline xml/security_ldap.xml]: Uhhhhh .... some things can be changed, some can't? I think instance name can never be changed post install. Suffix can be changed, but it's a big effort. Everything else is easy to change.

File: xml/security_ldap.xml (hunk 2 above)
Comment (taroth21) [reply xml/security_ldap.xml]: Yes, you are right - I tried on SLE 15 SP1 with the latest updates for 389-ds installed. I installed the latest snapshot for SLE 15 SP2 today and could finish the setup in YaST without certificates.

File: xml/security_ldap.xml (hunk 1 above)
Comment (taroth21) [reply xml/security_ldap.xml]: Changed that, thanks.

**Pull request merged**